src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -286,6 +288,12 @@ module.exports = function<T, P, I, TI, C, CX>(config : HostConfig<T, P, I, TI, C
   function commitAllLifeCycles() {
     while (nextEffect) {
       const current = nextEffect.alternate;
+
+      // Auto-focus host components once they have been mounted.
+      if (nextEffect.effectTag & Focus) {
+        focusNode(nextEffect.stateNode);
```

Comment:
Similarly this is a DOM-specific utility function and shouldn't get called in non-DOM environments.